README.md

```diff
@@ -3,19 +3,238 @@
+    satisfies_jurisdiction(input.app, cluster)
+    satisfies_pci(input.app, cluster)
+}
+
+satisfies_jurisdiction(app, cluster) {
+    not app.tags["requires-eu"]
+}
+
+satisfies_jurisdiction(app, cluster) {
+    app.tags["requires-eu"]
+    startswith(cluster.region, "eu-")
+}
+
+satisfies_pci(app, cluster) {
+    not app.tags["requires-pci-level"]
+}
+
+satisfies_pci(app, cluster) {
+    level = to_number(app.tags["requires-pci-level"])
+    level >= cluster.tags["pci-level"]
+}
+```
+
+#### Example Queries
+
+**Display all clusters:**
+
+```ruby
+> data.clusters[cluster_id]
++------------+------------------------------------------------+
+| cluster_id |           data.clusters[cluster_id]            |
++------------+------------------------------------------------+
+| "prod-eu"  | {"region":"eu-central","tags":{"pci-level":2}} |
+| "prod-us"  | {"region":"us-east"}                           |
+| "test-eu"  | {"region":"eu-west","tags":{"pci-level":4}}    |
+| "test-us"  | {"region":"us-west"}                           |
++------------+------------------------------------------------+
+```
+
+**Where will this app be deployed?**
+
+```ruby
+> input = {"app": {"tags": {"requires-pci-level": "3", "requires-eu": "true"}}}
+> app_placement
+[
+    "prod-eu"
+]
+```
+
+### Example: SSH Auditing
+
+This example shows how you can audit who has SSH access to hosts within
+different clusters. We will assume that SSH access is granted via group access
+in LDAP.
+
+```ruby
+import data.ldap
+import data.clusters
+
+ssh_access[[cluster_name, host_id, user_id]] {
+    host_id = clusters[cluster_name].hosts[_]
+    group_id = ldap.users[user_id].groups[_]
+    group_id = clusters[cluster_name].groups[_]
+}
+
+prod_users = {user_id | ssh_access[["prod", _, user_id]]}
+```
+
+#### Example Queries
+
+**Display all LDAP users:**
+
+```ruby
+> data.ldap.users[user_id]
++-------------------------------+---------+
+|   data.ldap.users[user_id]    | user_id |
++-------------------------------+---------+
+| {"groups":["dev","platform"]} | "alice" |
+| {"groups":["dev","ops"]}      | "bob"   |
+| {"groups":["dev"]}            | "janet" |
++-------------------------------+---------+
+```
+
+**Display all cluster/group pairs:**
+
+```ruby
+> data.clusters[cluster_id].groups[_] = group_id
++------------+------------+
+| cluster_id |  group_id  |
++------------+------------+
+| "test"     | "dev"      |
+| "test"     | "ops"      |
+| "prod"     | "ops"      |
+| "prod"     | "platform" |
++------------+------------+
+```
+
+**Display all cluster/host pairs:**
+
+```ruby
+> data.clusters[cluster_id].hosts[_] = host_id
++------------+------------+
+| cluster_id |  host_id   |
++------------+------------+
+| "prod"     | "host-123" |
+| "prod"     | "host-345" |
+| "prod"     | "host-567" |
+| "test"     | "host-abc" |
+| "test"     | "host-cde" |
+| "test"     | "host-efg" |
++------------+------------+
+```
+
+**Who can access production hosts?**
```

Comment:
I could see this being the first query.  (even though they might not understand the schema, which the other queries show.)